packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -69,16 +73,39 @@ export const usePeoplePane = (props: {
 
   const localeStrings = useLocale().strings.call;
 
+  /* @conditional-compile-remove(spotlight) */
+  const sidePaneHeaderMenuProps: IContextualMenuProps = useMemo(() => {
+    const menuItems: IContextualMenuItem[] = [];
+    if (onStopAllSpotlight && spotlightedParticipantUserIds && spotlightedParticipantUserIds.length > 0) {
+      menuItems.push({
```

Comment:
I'm curious as to what this looks like in the side pane, this doesn't affect the regular side pane only when in a interop meeting?

Reply:
Good question. The more button only shows up when someone in the meeting is spotlighted so it will appear in non-Teams calls. This is a screenshot of a group call:
![image](https://github.com/Azure/communication-ui-library/assets/79475487/dee2e844-5f0b-4b75-ac20-00e2fcc5f81c)
Please also see the attached videos :)